docs/guide/testing.md

```diff
@@ -50,7 +63,8 @@ describe('ExplicitReferenceComponent', () => {
 
 ## Test components using [icon library](../usage/icon-library.md)
 
-When it comes to testing components using icon library you'll need to setup the icon library in one way or another otherwise component will throw because of the missing icon definition. The icon library setup is needed because most likely `AppModule` where you normally add icons to the library is not part of the tested component spec and therefore icons added there are not available.
+When it comes to testing components using icon library you'll need to setup the icon library in one way or another otherwise component will throw because of the missing icon definition.
+The icon library setup is needed because most likely `AppModule` where you normally add icons to the library is not part of the tested component spec and therefore icons added there are not available.
 
 There are two main options how to deal with the icon library in the tests:
```

Comment:
Please add the example as another section (similar to those below). Currently this PR changes example code provided for a completely different way of using `angular-fontawesome` (explicit reference) and this is not right.